README.md

```diff
@@ -6,4 +6,6 @@ simulations by automating commonly performed tasks.
 
 See [documentation](https://e3sm-project.github.io/zppy) for more details.
 
-Run `python -m unittest tests/*.py` to run all tests.
\ No newline at end of file
+Run `python -m unittest discover -s tests -p "test_*.py` to run all tests.
+Run `python -m unittest tests/test_*.py"` to run only unit tests.
+Run `python -m unittest tests/integration/test_*.py"` to run only integration tests (on server)
```

Comment:
I get the following:
```
$ python -m unittest tests/integration/test_bash_generation.py
Validation results={'default': {'input': False, 'input_subdir': True, 'output': False, 'case': False, 'www': False, 'debug': True, 'partition': True, 'e3sm_unified': True, 'dry_run': True}, 'climo': True, 'ts': True, 'e3sm_diags': True, 'e3sm_diags_vs_model': True, 'amwg': True, 'mpas_analysis': True, 'global_time_series': True}
Traceback (most recent call last):
  File "/home/ac.forsyth2/miniconda3/envs/zppy_dev/bin/zppy", line 8, in <module>
    sys.exit(main())
  File "/home/ac.forsyth2/miniconda3/envs/zppy_dev/lib/python3.7/site-packages/zppy/__main__.py", line 33, in main
    raise Exception('Configuration file validation failed')
Exception: Configuration file validation failed
```
So, `input`, `output`, and `www` aren't passing validation because of this pull request. Those values in `test_bash_generation.cfg` are:
```
input = .
output = test_output
www = www/path
```
It's not clear to me why moving the test/config files to a subdirectory would invalidate  the config file.

Reply:
Never mind, the python file just needs to be updated: `self.assertEqual(os.system('zppy -c tests/integration/test_bash_generation.cfg'), 0)`

Reply:
Thanks, fixed this